Rewrite Game Modifiers + Create Mod Stubs (#3809)
ppy/osu-wiki: wiki/Game_modifier/Hidden/en.md

---
stub: true
tags:
  - hidden
  - HD
  - mod
  - game modifier
---

# Hidden (mod)

![HD mod icon](/wiki/shared/mods/HD.png "Hidden (HD) mod icon")

*For the full list of all mods, see: [Game Modifiers](/wiki/Game_modifier)*\
*Not to be confused with [Fade In (mod)](/wiki/Game_modifier/Fade_In) or [Flashlight (mod)](/wiki/Game_modifier/Flashlight).*

## About

- Abbreviation: HD
- Type: Difficulty Increasing

> - Score Multiplier: • ![][o!s] ![][o!t] ![][o!c]: 1.06x • ![][o!m]: 1.00x

Reviewer: ```suggestion
- Score Multiplier:
  - ![][o!s] ![][o!t] ![][o!c]: 1.06x
  - ![][o!m]: 1.00x
```

- Default shortcut key: `F`
  - Default shortcut key ([osu!mania](/wiki/Game_Modes/osu!catch)): `F` `F` or `Shift`+`F`

> - Caption: • ![][o!s]: `Play with no approach circles and fading notes for a slight score advantage.` • ![][o!t]: `The notes fade out before you hit them!` • ![][o!c] `Play with no approach circles and fading notes for a slight score advantage.` • ![][o!m]: `The notes fade out before you hit them!`

Reviewer: ```suggestion
- Caption:
  - ![][o!s]: `Play with no approach circles and fading notes for a slight score advantage.`
  - ![][o!t]: `The notes fade out before you hit them!`
  - ![][o!c] `Play with no approach circles and fading notes for a slight score advantage.`
  - ![][o!m]: `The notes fade out before you hit them!`
```

### osu!mania

In [osu!mania](/wiki/Game_Modes/osu!mania), the Hidden mod functions as the inverse of the Fade In mod as the notes, and thus, it will cause the notes  ot fade out before you are supposed to hit them.

![Gameplay of HD in osu!mania](img/GM_HD2_M.jpg "Screenshots of osu!mania gameplay with the Hidden mod, at different combo milestones")

## Trivia

- The Hidden mod debuted in Ouendan 2, which was the second Japanese DS game in the [Osu! Tatake! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia") series (the series that osu! is based on).
- If a beatmap is passed with an S or SS grade with the Hidden mod enabled, the beatmap will award the silver variant of the grade instead.
- By default, in [osu!standard](/wiki/Game_Modes/osu!) the first [hit object](/wiki/Hit_object)'s [approach circle](/wiki/Hit_object/Approach_circle) will be temporarily visible at the start of a map so as to help players better gauge when to tap said hit object. This may be turned off in the Options under `Gameplay`.
- In osu!mania, the Hidden mod is a variant of the [Fade In](/wiki/Game_modififer/Fade_In) mod.

> - The current version of the Hidden mod in osu!mania used to be a seperate mod called "Fade-out."

Reviewer: ```suggestion
- The current version of the Hidden mod in osu!mania used to be a separate mod called [Fade Out](/wiki/Game_modifier/Fade_Out).
```
